flang/lib/Optimizer/Analysis/AliasAnalysis.cpp

```diff
@@ -497,6 +482,29 @@ static Value getPrivateArg(omp::BlockArgOpenMPOpInterface &argIface,
   return privateArg;
 }
 
+/// Temporary function to skip through all the no op operations
```

Comment:
Can you not move and change the function at the same time? It is harder to see what the actual change is. If it is a function that is defined before, you can just provide a declaration instead.

Reply:
Done.